models/DataObject/Data/Link.php

```diff
@@ -283,7 +332,7 @@ public function getPath(): string
                 $path = $this->getElement()->getFullPath();
             }
         } else {
-            $path = $this->getDirect();
+            $path = $this->getDirect() ?? '';
```

Comment:
But `getDirect()` wasn't changed by this PR. It were a new BC break

Reply:
@kingjia90 It is enough to only make the property nullable. The admin interface only set empty strings to the setters. I think the null values are the result of imports, because the properties had no default value.

I changed it https://github.com/pimcore/pimcore/pull/15577/commits/bca8563ced16ff66d08faded084a2832152f0726

Reply:
"_The admin interface only set empty strings to the setters_" ~ not always true for `target` field as you can skip selecting the null value in data type "Allowed Targets" specific settings and this doesn't set the empty string anymore on opening the link editor and not selecting target:
<img width="241" alt="image" src="https://github.com/pimcore/pimcore/assets/5137917/e49d9ab3-882a-4ce4-a8a9-44074b010c3d">

Reply:
@dvesh3 Yes that is right. I changed it https://github.com/pimcore/pimcore/pull/15577/commits/f80554a7a3a8de73c5885f21fdd64f2dba52f2f6